Dockerfile.assisted-service

```diff
@@ -1,10 +1,33 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest as certs
+# Generate python client
+FROM swaggerapi/swagger-codegen-cli:2.4.15 as swagger_py
```

Comment:
Why do we need here all the tools for the build?
We don't use the build container anymore?

Reply:
Not for the production image. Look at the description of this PR.